features/step_definitions/sign_up_steps.rb

```diff
@@ -0,0 +1,11 @@
+When(/^I enter my email in the "([^"]*)" field$/) do |field_name|
+	fill_in "Email", with: :email
+end
+
+And(/^I enter my password in the "([^"]*)" field$/) do |field_name|
+	fill_in "Password", with: :password
+end
+
+And(/^I click the "([^"]*)" button$/) do |button|
+	click_button button
+end
```

Comment:
@tochman we are not finding this err, as Rubymine says it's already fixed. We'll try one more thing and see if we succeed.

![image](https://cloud.githubusercontent.com/assets/13518870/22300665/76808710-e328-11e6-9bbc-85f89dbc7bf5.png)